### DISCUSSION TITLE
Entrify command should generate a content migration

### DISCUSSION THREAD

**Discussion by Mosnar** [craftcms/cms#14660] [Ideas]
If I entrify a category group for example, I am ultimately prompted to run a command immediately after deployment to migrate the content in subsequent environments. This really isn't ideal since it means having to SSH into a server and potentially have a moment of broken content in production.

I think it would make more sense if the entrify command ultimately generated a content migration that could be run automatically (or in the CMS).

**Comment by brandonkelly** [comment]
Agree, we actually talked about this internally a bit ago, and it’s something I want to look into as a potential improvement :)
